schema/v0.7/schema.json

```diff
@@ -0,0 +1,243 @@
+{
+    "definitions": {},
+    "$schema": "http://json-schema.org/draft-04/schema#", 
+    "$id": "https://example.com/object1595636524.json", 
+    "title": "Root", 
+    "type": "object",
+    "required": [
+        "nodeList"
+    ],
+    "properties": {
+        "nodeList": {
+            "$id": "#root/nodeList", 
+            "title": "Nodelist", 
+            "type": "array",
+            "default": [],
+            "items":{
+                "$id": "#root/nodeList/items", 
+                "title": "Items", 
+                "type": "object",
+                "required": [
+                    "altitude",
+                    "dateAdded",
+                    "group",
+                    "ipv4",
+                    "latitude",
+                    "longitude",
+                    "name",
+                    "type",
+                    "status"
+                ],
+                "properties": {
+                    "altitude": {
+                        "$id": "#root/nodeList/items/altitude", 
+                        "title": "Altitude", 
+                        "type": "integer",
+                        "examples": [
+                            170
+                        ],
+                        "default": 0
+                    },
+                    "antennaCone": {
+                        "$id": "#root/nodeList/items/antennaCone", 
+                        "title": "Antenna Cone", 
+                        "type": "integer",
+                        "examples": [
+                            120
+                        ],
+                        "default": 0
+                    },
+                    "antennaDirection": {
+                        "$id": "#root/nodeList/items/antennaDirection", 
+                        "title": "Antenna Direction", 
+                        "type": "integer",
+                        "examples": [
+                            200
+                        ],
+                        "default": 0
+                    },
+                    "antennaDistance": {
+                        "$id": "#root/nodeList/items/antennaDistance", 
+                        "title": "Antenna Distance", 
+                        "type": "integer",
+                        "examples": [
+                            2500
+                        ],
+                        "default": 0
+                    },
+                    "antennaModel": {
+                        "$id": "#root/nodeList/items/antennaModel", 
+                        "title": "Antenna Model", 
```

Comment:
```suggestion
                        "title": "Model Number", 
```

Reply:
I changed this to `Hardware Model`, if that's good.